tools/ios/assemble_pod_package.py

```diff
@@ -81,16 +81,25 @@ def assemble_pod_package(
     pod_name = "onnxruntime-extensions-c"
     pod_summary = "ONNX Runtime Extensions C/C++ Pod"
     pod_description = "Pod containing pre and post processing custom ops for onnxruntime."
-    with open(framework_info_file, mode="r") as f:
-        framework_info = json.load(f)
-        ios_deployment_target = framework_info["apple_deployment_target"]
+    with open(xcframework_info_file, mode="r") as f:
+        xcframework_info = json.load(f)
+        ios_deployment_target = ""
```

Comment:
```suggestion
        print(f"Setting deployment target from available frameworks: {', '.join(xcframework_info.keys())}")
        ios_deployment_target = ""
```

To aid debugging if there's an issue with an empty deployment target value later on it might help to output which frameworks were available here.